bfvmm/include/test/hve.h

```diff
@@ -172,10 +172,12 @@ setup_vcpu(MockRepository &mocks, ::intel_x64::vmcs::value_type reason = 0)
     mocks.OnCall(vcpu, bfvmm::intel_x64::vcpu::execute_rdcr3);
     mocks.OnCall(vcpu, bfvmm::intel_x64::vcpu::execute_wrcr3);
     mocks.OnCall(vcpu, bfvmm::intel_x64::vcpu::execute_wrcr4);
-    mocks.OnCall(vcpu, bfvmm::intel_x64::vcpu::add_cpuid_handler);
-    mocks.OnCall(vcpu, bfvmm::intel_x64::vcpu::add_cpuid_emulator);
-    mocks.OnCall(vcpu, bfvmm::intel_x64::vcpu::execute_cpuid);
-    mocks.OnCall(vcpu, bfvmm::intel_x64::vcpu::enable_cpuid_whitelisting);
```

Comment:
I removed the enable_whitelisting function from the cpuid interface per a discussion with Rian the other day. Let me know if I misinterpreted our conversation...

Reply:
The white list function should be removed. The other functions should not be.